features/environment.py

```diff
@@ -0,0 +1,18 @@
+from behave import *
+import ModelController
+
+def before_all(context):
+    context.MEPath = "/home/seanjkeene/PycharmProjects/doe-egot-me/"
+
+    print("First run...")
+    try:
+        ModelController.main(test_mode=False, DERSHDI_FilePath=r"DERSHistoricalData Inputs/TP_ME1_A_LogInput.csv")
+    except SystemExit:
+        context.firstfilename = ModelController.mcConfiguration.output_log_name
```

Comment:
This could be a pass function, but it's a convenient place to add the output log filenames to the context object (which is important since they're generated based on realtime, so they need to be available to the steps.py in order to verify that the logs we are testing are the same logs that were generated by the tests invoked by behave.